core/client/hdfs/src/test/java/alluxio/hadoop/HadoopConfigurationUtilsTest.java

```diff
@@ -39,19 +41,20 @@ public void after() {
    * configuration.
    */
   @Test
-  public void mergeEmptyHadoopConfiguration() {
+  public void mergeEmptyHadoopConfiguration() throws URISyntaxException {
```

Comment:
why throw this?

Reply:
Removed, there is no that exception thrown here.